dali/kernels/imgproc/warp/warp_uniform_size_impl.cuh

```diff
@@ -31,17 +31,25 @@ __global__ void BatchWarpUniformSize(
     const SampleDesc<ndim, OutputType, InputType> *samples,
     ivec<ndim> output_size,
     ivec<ndim> block_size,
+    int z_blocks_per_sample_shift,
     const mapping_params_t<Mapping> *mapping,
     BorderType border) {
   BlockDesc<ndim> block;
-  block.sample_idx = blockIdx.z;
-  block.start = 0;
-  block.start.x = blockIdx.x * block_size.x;
-  block.start.y = blockIdx.y * block_size.y;
+  ivec<ndim> block_idx(0);
+  block_idx.x = blockIdx.x;
+  block_idx.y = blockIdx.y;
```

Comment:
Can we assume that `ndim > 2` and we have `block_idx.y` and `block_idx[2]`?

Reply:
No, we can't.